integration/kubernetes/helm-chart/alluxio/CHANGELOG.md

```diff
@@ -85,3 +85,7 @@
 0.6.1
 
 - Infer hostNetwork, dnsPolicy and domain socket volume type based on the user
+
+0.6.2
+
+- Fix alluxio chart failed to deploy with helm when "fuse.enabled" is true in values.yaml(issue: #11542)
```

Comment:
I think "Fix podAffinity for fuse DaemonSet" is better

Reply:
You can leave the url there for the issue tracker like "https://github.com/Alluxio/alluxio/issues/11542"